pkg/kubelet/status/manager.go

```diff
@@ -285,17 +255,28 @@ func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 		return
 	}
 
+	// Get the pod name & namepsace.
+	var name, namespace, loggedName string
+	if cachedPod, ok := m.podManager.GetPodByUID(uid); ok {
```

Comment:
The pod UID, name, and namespace don't change.I think there is value of caching that, instead of getting the pod from the manager again, which requires acquiring the lock again. What do you think?